store/rootmulti/store.go

```diff
@@ -827,7 +827,6 @@ func (rs *Store) Snapshot(height uint64, protoWriter protoio.Writer) error {
 				node, err := exporter.Next()
 				if err == iavltree.ErrorExportDone {
 					rs.logger.Debug("snapshot Done", "store", store.name, "nodeCount", nodeCount)
-					nodeCount = 0
```

Comment:
This is getting removed because it's right before a `break` and then we don't use `nodeCount` outside of the for loop anymore?